docs/wiki/GettingStarted.md

```diff
@@ -167,3 +172,44 @@ Last but not least, instead of fetching your own copy of the repository you can
 <!-- Docs -->
 [PowerShellDocs]: <https://docs.microsoft.com/en-us/powershell/>
 [AzureNames]: <https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules>
+
+### Parameter File Tokens
+
+If you are forking or cloning the repository. Start by leveraging tokens in your parameter files. Tokens allow you to test in isolation, while also leveraging your own naming conventions when deploying resources, or other customizations to your own resources. Details are in the [Parameter File Tokens Documentation](./ParameterFileTokens.md).
+
+The repository contains a [Settings JSON](../../settings.json) that has a default token called `namePrefix`. You can use a custom value here and then start adding the tokens in your parameter files. The tokens default format is `<<tokenName>>` (i.e. `<<namePrefix>>`). Here is an example token:
```

Comment:
Relative links only works when its relative within the docs folder when this is published into the wiki. Use public link.

Reply:
Fixed

Reply:
This is not fixed, still says: "\[Settings JSON\]\(../../settings.json\)". When this is publised, the link is no longer true.

Reply:
Sorry I missed it as its in another file. Should be fixed now